xml/System.Diagnostics/Process.xml

```diff
@@ -1033,7 +1033,7 @@ The following code example creates a process that prints a file. It sets the <xr
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Diagnostics.Process.ErrorDataReceived> event indicates that the associated process has written to its redirected <xref:System.Diagnostics.Process.StandardError%2A> stream.  
+ The <xref:System.Diagnostics.Process.ErrorDataReceived> event indicates that the associated <xref:System.Diagnostics.Process> has written a line, terminating with a newline (CR, LF or CR+LF), to its redirected <xref:System.Diagnostics.Process.StandardError%2A> stream.   
```

Comment:
```suggestion
 The <xref:System.Diagnostics.Process.ErrorDataReceived> event indicates that the associated <xref:System.Diagnostics.Process> has written a line that's terminated with a newline (`CR`, `LF`, or `CR+LF`) to its redirected <xref:System.Diagnostics.Process.StandardError%2A> stream.   
```

Reply:
Why should CR and LF be displayed as code? It's not code in some programming language (like e.g. `\r\n` or `vbCrLf`), they are abbreviations and I think those should be shown as regular text.

Reply:
I made a different suggestion. Feel free to accept it or not. I would at least accept the "that's terminated" part though.

Reply:
@gewarren Yeah, that sounds good to me, accepted.